### PR TITLE
Add Travel and Accommodation policy

### PR DESCRIPTION
**Pull request by dragon-dxw**

### CODE REVIEW

File: _includes/sections/working-here/managing-your-day-to-day-work.md [hunk 1]
```diff
@@ -158,6 +158,39 @@ incurring them.
...
+
+A business critical need is defined as:
+
+- Client meetings
+- Meetings with your line manager (e.g. during probation, monthly check-ins,
+  quarterly reviews, post parental / sick leave)
+- Team meetups (e.g. design community, tech forum, away days, Board meetings)
+- All company in-person events
+- Conferences and training events
+- At present, there is no obligation to attend in-person meetings.
```
Comment (johnwaterworth) [inline _includes/sections/working-here/managing-your-day-to-day-work.md:179]: This isn't part of the bullet list. It should be a paragraph on its own.

File: _includes/sections/working-here/managing-your-day-to-day-work.md [hunk 2]
```diff
@@ -158,6 +158,39 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+#### Travel and Accommodation Policy
+
+Currently the default for everyone in the organisation is working from home
```
Comment (johnwaterworth) [inline _includes/sections/working-here/managing-your-day-to-day-work.md:163]: This is contradictory. I suggest:
Currently, the default for everyone in the organisation is working from home. For this period, we will consider regular travel to an office space without a business critical need as commuting.

**Review by johnwaterworth — CHANGES_REQUESTED** [review]
A couple of small edits needed.

File: _includes/sections/working-here/managing-your-day-to-day-work.md [hunk 3]
```diff
@@ -158,6 +158,39 @@ incurring them.
...
+Commuting costs are the responsibility of individuals, while dxw will fund
+business critical travel and accommodation.
+
+A business critical need is defined as:
+
+- Client meetings
+- Meetings with your line manager (e.g. during probation, monthly check-ins,
+  quarterly reviews, post parental / sick leave)
+- Team meetups (e.g. design community, tech forum, away days, Board meetings)
+- All company in-person events
```
Comment (dragon-dxw) [inline _includes/sections/working-here/managing-your-day-to-day-work.md:177]: Potentially ambiguous: could be construed as "all-company events" i.e. things where everyone meets up, which I think is not the intention.

Comment (johnwaterworth) [reply _includes/sections/working-here/managing-your-day-to-day-work.md:177]: Given that the list mentions individual meetings, then team meetings, I suspect that this bullet is about events that are for everyone in dxw.

So maybe:
 - All-staff in-person events

**Review by johnwaterworth — APPROVED** [review]
Looks good to go.

**Pull request merged**